app/config/settings.py

```diff
@@ -87,6 +87,7 @@ def strtobool(val) -> bool:
 ANONYMOUS_USER_NAME = "AnonymousUser"
 REGISTERED_USERS_GROUP_NAME = "__registered_users_group__"
 REGISTERED_AND_ANON_USERS_GROUP_NAME = "__registered_and_anonymous_users__"
+CHALLENGE_REVIEWERS_GROUP_NAME = "challenge_reviewers"
```

Comment:
I would prefer not to add any new special groups. If you need this group in the tests then create a pytest fixture for it. We should then leave it up to the site admins to create groups that are people centric, and attach the permissions to those.